drivers/devmapper/deviceset.go

```diff
@@ -2690,8 +2692,6 @@ func NewDeviceSet(root string, doInit bool, options []string, uidMaps, gidMaps [
 			devices.filesystem = val
 		case "dm.mkfsarg":
 			devices.mkfsArgs = append(devices.mkfsArgs, val)
-		case "dm.mountopt":
-			devices.mountOptions = joinMountOptions(devices.mountOptions, val)
```

Comment:
I get that the `devicemapper.mountopt` name comes from the driver name and how the option is parsed from the config file, but removing recognition of `dm.mountopt` seems unnecessary.  Can we treat the two names as values for the same case?

Reply:
Wouldn't this just confuse people?  I don't see a reason to have two options to do the same thing.

Reply:
They do the same thing.  Otherwise we're renaming it, and configurations that used one name for the option will start returning an error in anything that uses the newer version of this library.

Reply:
Added back with no documentation.